eng/Subsets.props

```diff
@@ -218,15 +220,30 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('-depproj-'))">
-    <DepprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.depproj" SignPhase="R2RBinaries" BuildInParallel="false" />
+    <DepprojProjectToBuild Condition="'$(RuntimeFlavor)' == 'CoreCLR'" Include="$(InstallerProjectRoot)pkg\projects\**\*.depproj" SignPhase="R2RBinaries" BuildInParallel="false" />
+    <DepprojProjectToBuild Condition="'$(RuntimeFlavor)' == 'Mono'" Include="$(InstallerProjectRoot)pkg\projects\**\*.depproj" SignPhase="Binaries" BuildInParallel="false" />
     <ProjectToBuild Include="@(DepprojProjectToBuild)" />
   </ItemGroup>
 
-  <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('-pkgproj-'))">
+  <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('-pkgproj-')) and '$(TargetOS)' != 'iOS'">
     <PkgprojProjectToBuild Include="$(InstallerProjectRoot)pkg\projects\**\*.pkgproj" SignPhase="MsiFiles" BuildInParallel="false" />
     <ProjectToBuild Include="@(PkgprojProjectToBuild)" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subsetCategory.Contains('installer')) and $(_subset.Contains('-pkgproj-')) and '$(TargetOS)' == 'iOS'">    
+    <!-- For iOS we only need to construct runtime packs-->
+    <PkgprojProjectToBuild  
+        Include="$(InstallerProjectRoot)pkg\projects\netcoreapp\pkg\Microsoft.NETCore.App.Runtime.pkgproj" 
+        SignPhase="MsiFiles" BuildInParallel="false" />
+    <PkgprojProjectToBuild  
```

Comment:
Could you add comment what legacy and workaround is about?

Reply:
Those are pre-existing projects/directories but yeah it'd be good to find out what they're used for and whether we can simplify.

Reply:
We may not need it if I can handle it at the pkgproj level.

@dagood Do you know the history?

Reply:
Legacy produces the old nupkgs that formed NETCoreApp in 2.1. Workaround produces a package to work around some issues that were found in 3.0 SDK development. https://github.com/dotnet/core-setup/pull/6096, https://github.com/dotnet/runtime/issues/3528.

But yes, I do not think they should be called out explicitly in this file. Adding a readme to the project dir or inside the project files would be the way to go IMO.

Reply:
> whether we can simplify.

Probably. However, the Host tests (`src/installer/test/Assets`) currently still depend on the legacy packages. The legacy packages were also used for testing by downstream repos even after 3.0, and may still be.